tools/ui-components/src/form-group/form-group.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { FormGroup, FormGroupProps } from '.';
+
+const story = {
+  title: 'Example/FormGroup',
+  component: FormGroup,
+  argTypes: {
+    children: { control: { type: 'object' } },
```

Comment:
I wanted to create multiple children and can test them, but reading https://storybook.js.org/docs/react/essentials/controls I don't think it's possible unless if I create custom controls.

So I think I will create the whole form and create a custom controls with that form

Reply:
When we have all the form components, we can nest in formGroup story (unless we do the research and come up with another pattern).

Reply:
I am growing Ahmad 🤣, I have figured out a simple way to do it 🤣